juriscraper/opinions/united_states/state/nd.py

```diff
@@ -50,11 +54,24 @@ def _process_html(self) -> None:
 
             for idx, txt in enumerate(raw_values[:5]):
                 if idx == 0:
+                    # Separate case name and citation if present
+                    match = re.match(
+                        r"^(.*?)(\s*((\d{4}\sND\s\d+)|(1 \d\.N\.W\d d\d+)))?\s*$",
+                        txt,
+                    )
+                    if match:
+                        case_name = match.group(1).strip()
+                        citation = (
+                            match.group(2).strip() if match.group(2) else ""
+                        )
+                        txt = case_name
+                    else:
+                        citation = ""
                     txt, _ = self.clean_name(txt)
```

Comment:
whats with returning other dockets in clean name?  we dont use it.  so whats the deal can we clean up the function